### DISCUSSION TITLE
Anyone else had problems with uncrustify?

### DISCUSSION THREAD

**Discussion by rempas** [w0rp/ale#3884] [Q&A - Ask for help with problems]
I actually opened an [issue](https://github.com/dense-analysis/ale/issues/3882) describing the problem but TLDR, **uncrustify** doesn't get recognized from ALE while of course it is installed. Anyone else had these problem?

**Comment by hsanson** [comment]
This tool is a fixer, not a linter.  You have to invoke it using either `:ALEFix` or `:ALEFix uncrustify`.
You can also configure ALE to run all available fixers on save by setting `g:ale_fix_on_save`.
